code/script.js

```diff
@@ -1,47 +1,110 @@
+// FUNCTIONS:
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// 1rs Function showMessage: This function will add a chat bubble in the correct place based on who the sender is.
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="./Pictures/bill-chatbot.png" alt="User" />  
       </section>
-    `
-  } else if (sender === 'bot') {
+    `;
+  } else if (sender === "bot") {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="./Pictures/julieta-chatbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+  chat.scrollTop = chat.scrollHeight; // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+};
 
-// Starts here
+// 2nd Function greeting: This function invokes ('calls') the first function, so when the function greeting is invoked, it invokes the first function.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Welcome to Flowerland! What's your name?`, "bot");
+};
+setTimeout(greeting, 1000);
+
+// 3rd Function handleInput
+const handleInput = (event, answer) => {
+  event.preventDefault();
+
+  if (currentQuestion === 1) {
+    let name = nameInput.value;
+    showMessage(name, "user");
+    setTimeout(() => showFlowerTypes(name), 1000);
+
+  } else if (currentQuestion === 2) {
+    showMessage(answer, "user");
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showFlowerSize(answer), 2000);
+
+  } else if (currentQuestion === 3) {
+    showMessage(answer, "user" );
+    setTimeout(()=> showFinalMessages(answer),2000);
+  }
+};
+
+// THREE FUNCTIONS THAT ARE BEING INVOKED INSIDE OF THE HANDLEINPUT FUNCTION:
+
+function showFlowerTypes(name) {
+  showMessage(`What type of flowers would you like to order ${name}?`, "bot");
+  inputWrapper.innerHTML = 
+  `
+  <select id="select">
+    <option value="" selected disabled>👇 Select a bouquet of flowers...</option>
+    <option value="Rose">Roses</option>
+    <option value="Tulip">Tulips</option>
+    <option value="Sunflower">Sunflowers</option>
+    <option value="Magnolia">Magnolia</option>
+  </select>
+`;
+  const select = document.getElementById("select");
+  select.addEventListener("change", (event) => handleInput(event, select.value));
+  currentQuestion++;
+}
+
+function showFlowerSize(answer) {
```

Comment:
nice, that you have used the functional programming approach and created separate functions to handle each question.